### PR TITLE
i18n(zh-cn): Translate GitHub Pages and Vercel deploy guides

### PR DESCRIPTION
**Pull request by mogeko**
#### What kind of changes does this PR include?

- Translated content

#### Description

A Chinese translation about **how to deploy your Astro site to the web using GitHub Pages**.

Here is the corresponding English page for this translation: [Deploy your Astro Site to GitHub Pages](https://docs.astro.build/en/guides/deploy/github/)

And it is been marked `missing` in the #438.

### CODE REVIEW

**Comment by mogeko** [comment]
Update a new translated content!

This time it's a relevant translation **deployed to Vercel**.

Here is the corresponding English page for this translation: [Deploy your Astro Site to Vercel](https://docs.astro.build/en/guides/deploy/vercel)

And it is been marked missing in the https://github.com/withastro/docs/issues/438.

**Comment by mogeko** [comment]
I saw news about hacktoberfest in the Discord channel. So, why not?

If everything is ok with the PR, I would appreciate it beign labeled with `hacktoberfest-accepted `!

Thank u.😃

File: src/pages/zh-cn/guides/deploy/vercel.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+### 静态站点
+
+默认情况下，你的 Astro 项目是一个静态站点。你无需任何额外配置即可将静态 Astro 站点部署到 Vercel。
+
+:::note
+目前在 Vercel 上显示 Astro 网站的 404 页面存在一些问题。在解决这些问题之前，你需要在项目的根目录中添加以下配置文件：
+
+```json title="vercel.json"
+{
+  "cleanUrls": true,
```
Comment (delucis) [inline src/pages/zh-cn/guides/deploy/vercel.md:24]: JSON syntax error

```suggestion
  "cleanUrls": true
```

File: src/pages/zh-cn/guides/deploy/vercel.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+在你的项目完成导入和部署后，所有后续内容推送到（生产分支外的）分支都将自动生成 [预览部署（Preview Deployments）](https://vercel.com/docs/concepts/deployments/environments#preview)，以及对生产分支（通常是名为“main”的分支）所做的任何更改都将自动被部署为 [生产部署（Production Deployment）](https://vercel.com/docs/concepts/deployments/environments#production)。
+
+📚 详细请参考 Vercel 文档的 [Git 集成](https://vercel.com/docs/concepts/git)部分。
+
+### 通过 CLI 部署
+
+1. 安装 [Vercel CLI](https://vercel.com/cli) 并运行 `vercel` 以进行部署。
+
+    ```bash
+    npm i -g vercel
```
Comment (delucis) [inline src/pages/zh-cn/guides/deploy/vercel.md:80]: We switched to the full `install` command in the English version recently

```suggestion
    npm install -g vercel
```

**Review by delucis — COMMENTED** [review]
Going to make a couple of small fixes before merging.

**Review by delucis — APPROVED** [review]
Thanks for these helpful additions @mogeko!

**Comment by mogeko** [comment]
thanks for your suggestion @delucis

**Pull request merged**